dcpquery/db/__main__.py

```diff
@@ -17,6 +17,7 @@
 parser.add_argument("--dry-run", action="store_true", help="Print commands that would be executed without running them")
 args = parser.parse_args(sys.argv[1:])
 
+# Todo why is this in two places?
```

Comment:
TODO and FIXME comments should have an assignee. Can I help figure out what the issue is?

Reply:
discussed in person